tests/unit/rpc/handlers/LedgerEntryTests.cpp

```diff
@@ -2612,3 +2614,239 @@ TEST(RPCLedgerEntrySpecTest, DeprecatedFields)
     EXPECT_EQ(warning.at("id").as_int64(), static_cast<int64_t>(rpc::WarningCode::warnRPC_DEPRECATED));
     EXPECT_NE(warning.at("message").as_string().find("Field 'ledger' is deprecated."), std::string::npos) << warning;
 }
+
+// Same as BinaryFalse with include_deleted set to true
+// Expected Result: same as BinaryFalse
+TEST_F(RPCLedgerEntryTest, BinaryFalseIncludeDeleted)
+{
+    static auto constexpr OUT = R"({
+        "ledger_hash":"4BC50C9B0D8515D3EAAE1E74B29A95804346C491EE1A95BF25E4AAB854A6A652",
```

Comment:
I'd appreciate if you format the json a bit more by adding spaces between `:` and values. I know not all jsons in our tests are already formatted correctly - "thanks" to everyone using copilot 😢 
But let's try not to add more to the mess 😃